web/styles/left_sidebar.css

```diff
@@ -687,8 +673,8 @@ li.active-sub-filter {
     .unread_count {
         visibility: hidden;
     }
-    /* ...unless it's the selected home view. */
-    &.selected-home-view .unread_count {
+    /* ...unless it's the first-child which is always the selected home view. */
+    &:first-child .unread_count {
```

Comment:
I worry that the `:first-child` here might be brittle. I'm not sure. Kind of makes me second-guess the removal of the `.selected-home-view` class. Perhaps that should be added to the selected home view for exactly this kind of purpose?

Reply:
what is brittle about `:first-child` ? the home is always guaranteed to be the first `<li>` child

Reply:
It's absolutely not guaranteed. If some other element were introduced there, as can happen when new features are introduced, this would break.

Reply:
So you mean some new element may be introduced as the first child but (for some reason) not being the home view ?

Reply:
As I said, the whole approach depends on the home-view being the first child, so if this `first-child` property is not guaranteed it means that this whole approach is not reliable.

Reply:
We can also be more specific with:

```css
#left-sidebar-navigation-list > li:first-child .unread_count {
    visibility: visible;
}
```

Reply:
I guess, but why is that better than just continuing to add and remove a `.selected-home-view` class? And referencing that?